sdk/core/Azure.Core/src/Shared/ClientDiagnostics.cs

```diff
@@ -49,7 +49,7 @@ public ClientDiagnostics(ClientOptions options, bool? suppressNestedClientActivi
         ///  The default value could change in the future, the flag should be only set to false if suppression for the client
         ///  should never be enabled.</param>
         public ClientDiagnostics(string optionsNamespace, string? providerNamespace, DiagnosticsOptions diagnosticsOptions, bool? suppressNestedClientActivities = null)
-            : base(optionsNamespace, providerNamespace, diagnosticsOptions.IsDistributedTracingEnabled, suppressNestedClientActivities.GetValueOrDefault(false))
+            : base(optionsNamespace, providerNamespace, diagnosticsOptions.IsDistributedTracingEnabled, suppressNestedClientActivities.GetValueOrDefault(true))
```

Comment:
Curiosity question: how are we validating that changing a default won't make breaking behavioral changes?  (I trust you, @lmolkova, I am primarily curious!)

Reply:
this is a behavior-breaking that we want to do. It's documented in the changelog as one.
We've been postponing this change till we are ready to stabilize tracing stuff and we now are.